src/main/scala/org/apache/mesos/chronos/scheduler/config/StatsPublisherConfiguration.scala

```diff
@@ -0,0 +1,14 @@
+package org.apache.mesos.chronos.scheduler.config
+
+import org.rogach.scallop.ScallopConf
+
+trait StatsPublisherConfiguration extends ScallopConf{
+  lazy val webhookUrl = opt[String]("stats_webhook_url",
+    descr = "Url to which tasks statistics are pushed",
```

Comment:
If for a store requiring authentication such as S3 where will the credentials be read from?